lib/server/controllers/gaqDetectors.controller.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const Joi = require('joi');
+const { DtoFactory } = require('../../domain/dtos/DtoFactory.js');
+const { dtoValidator } = require('../utilities/dtoValidator.js');
+const { updateExpressResponseFromNativeError } = require('../express/updateExpressResponseFromNativeError.js');
+const { gaqDetectorService } = require('../services/gaq/GaqDetectorsService.js');
+
+// eslint-disable-next-line valid-jsdoc
+/**
+ *
+ */
+const setGaqDetectors = async (req, res) => {
```

Comment:
Not following controller conventions. Please put what it does in the comment, and suffix the function by `Handler`

Reply:
applied